src/data.jl

```diff
@@ -501,6 +501,24 @@ end
 # Getting an index produces a tuple of the energy, 
 Base.getindex(d::DensityOfStates, ind) = (d.energy[ind], d.dos[ind], d.int[ind])
 
+# Setting up for gaussian smearing of DOS curve
+# Fourier transformed gaussian for convolution in smearing
+function Gaussian(
```

Comment:
Use the correct case for functions. Those that begin with capital letters and use PascalCase are related to structs (either declarations or constructors). For things that perform actions but aren't an explicit constructor, use snake_case.